@navikt/core/css/darkside/typography.darkside.css

```diff
@@ -1,238 +1,232 @@
-/* New in darkside */
+/* stylelint-disable csstools/value-no-unknown-custom-properties */
+
+/* ------------------------- Global typo-definitions ------------------------ */
```

Comment:
Just an experiment with section dividers here. Often feel i get lost in more complex components